charts/forms-flow-analytics/templates/deployment.yaml

```diff
@@ -17,281 +18,179 @@ spec:
-          valueFrom:
-            configMapKeyRef:
-              key: REDASH_DATABASE_URL
-              name: "{{ .Chart.Name }}"
-        {{- end }}
-        - name: WORKERS_COUNT
-          value: "1"
-        image: "{{ .Values.redash.image.registry}}/{{ .Values.redash.image.repository }}:{{ .Values.redash.image.tag }}"
-        name: "{{ .Chart.Name }}-scheduled-worker"
-        resources:
-          limits:
-            cpu: "{{ .Values.resources.limits.cpu }}"
-            memory: "{{ .Values.resources.limits.memory }}"
-          requests:
-            cpu: "{{ .Values.resources.requests.cpu }}"
-            memory: "{{ .Values.resources.requests.memory }}"
       restartPolicy: Always
-      imagePullSecrets:
-      - name: "{{ .Values.formsflow.auth}}"
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  labels:
-    app: "{{ .Chart.Name }}-scheduler"
-  name: "{{ .Chart.Name }}-scheduler"
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: "{{ .Chart.Name }}-scheduler"
-  strategy:
-  template:
-    metadata:
-      labels:
-        app: "{{ .Chart.Name }}-scheduler"
-    spec:
+      securityContext:
+        fsGroup: 2000 # Setting the file system group
       containers:
-      - args:
-        - scheduler
-        envFrom:
-        - configMapRef:
-            name: "{{ .Chart.Name }}" # default redash environment variables
-        - secretRef:
-            name: "{{ .Chart.Name }}" # default redash environment variables
-        env:
-        {{- if .Values.externalDatabase.existingDatabaseUrlKey }}
-        - name: REDASH_DATABASE_URL
-          valueFrom:
-            secretKeyRef:
-              key: {{ .Values.externalDatabase.existingDatabaseUrlKey }}
-              name: "{{ .Values.externalDatabase.existingSecretName }}"
-        {{- end }}
-        {{- if not .Values.externalDatabase.existingDatabaseUrlKey }}
-        - name: REDASH_DATABASE_URL
-          valueFrom:
-            configMapKeyRef:
-              key: REDASH_DATABASE_URL
-              name: "{{ .Chart.Name }}"
-        {{- end }}
-        image: "{{ .Values.redash.image.registry}}/{{ .Values.redash.image.repository }}:{{ .Values.redash.image.tag }}"
-        name: "{{ .Chart.Name }}-scheduler"
-        imagePullPolicy: Always
-        resources:
-          limits:
-            cpu: "{{ .Values.resources.limits.cpu }}"
-            memory: "{{ .Values.resources.limits.memory }}"
-          requests:
-            cpu: "{{ .Values.resources.requests.cpu }}"
-            memory: "{{ .Values.resources.requests.memory }}"
-      restartPolicy: Always
-      imagePullSecrets:
-      - name: "{{ .Values.formsflow.auth}}"
+        - name: '{{ .Chart.Name }}-scheduler'
+          image: '{{ .Values.redash.image.registry }}/{{ .Values.redash.image.repository }}:{{ .Values.redash.image.tag }}'
+          imagePullPolicy: Always
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 1000
```

Comment:
May need to check with @amal-k-soman-aot if this would work in openshift. I remember openshift having trouble in setting runAsUser. So we may need an option to not provide this and use whatever is default